Rename to minizip2. It seems to be agreed that this is what package managers
should do (so that we can offer minizip and minizip2 at the same time, since
they are not compatible) and yet upstream does not provide a way to do it.

I hope I've renamed everything I was supposed to rename.

https://github.com/nmoinvaz/minizip/issues/333
--- CMakeLists.txt.orig	2019-04-08 15:26:32.000000000 -0500
+++ CMakeLists.txt	2019-04-28 16:39:33.000000000 -0500
@@ -52,9 +52,9 @@
 set(INSTALL_INC_DIR ${CMAKE_INSTALL_FULL_INCLUDEDIR} CACHE PATH "Installation directory for headers")
 set(INSTALL_MAN_DIR ${CMAKE_INSTALL_FULL_MANDIR} CACHE PATH "Installation directory for manual pages")
 set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_FULL_LIBDIR}/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
-set(INSTALL_CMAKE_DIR "${CMAKE_INSTALL_FULL_LIBDIR}/cmake/minizip" CACHE PATH "Installation directory for cmake files.")
+set(INSTALL_CMAKE_DIR "${CMAKE_INSTALL_FULL_LIBDIR}/cmake/minizip2" CACHE PATH "Installation directory for cmake files.")
 
-set(MINIZIP_PC ${CMAKE_CURRENT_BINARY_DIR}/minizip.pc)
+set(MINIZIP_PC ${CMAKE_CURRENT_BINARY_DIR}/minizip2.pc)
 configure_file(minizip.pc.cmakein ${MINIZIP_PC} @ONLY)
 
 # Check if zlib installation is present
@@ -580,6 +580,7 @@
 set_target_properties(${PROJECT_NAME} PROPERTIES
                         VERSION ${VERSION}
                         SOVERSION ${SOVERSION}
+                        OUTPUT_NAME ${PROJECT_NAME}2
                         LINKER_LANGUAGE C
                         DEFINE_SYMBOL "MZ_EXPORTS"
                         POSITION_INDEPENDENT_CODE 1)
@@ -643,7 +644,7 @@
 # Build test executable
 if(MZ_BUILD_TEST)
     add_executable(minizip_cmd "minizip.c" "test/test.c" "test/test.h")
-    set_target_properties(minizip_cmd PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
+    set_target_properties(minizip_cmd PROPERTIES OUTPUT_NAME ${PROJECT_NAME}2)
     target_link_libraries(minizip_cmd ${PROJECT_NAME})
 
     if(NOT SKIP_INSTALL_BINARIES AND NOT SKIP_INSTALL_ALL)
--- minizip.c.orig	2019-04-08 15:26:32.000000000 -0500
+++ minizip.c	2019-04-29 16:21:26.000000000 -0500
@@ -62,14 +62,14 @@
 
 int32_t minizip_banner(void)
 {
-    printf("Minizip %s - https://github.com/nmoinvaz/minizip\n", MZ_VERSION);
+    printf("Minizip2 %s - https://github.com/nmoinvaz/minizip\n", MZ_VERSION);
     printf("---------------------------------------------------\n");
     return MZ_OK;
 }
 
 int32_t minizip_help(void)
 {
-    printf("Usage : minizip [-x -d dir|-l|-e] [-o] [-c codepage] [-a] [-j] [-0 to -9] [-b|-m] [-k 512] [-p pwd] [-s] file.zip [files]\n\n" \
+    printf("Usage : minizip2 [-x -d dir|-l|-e] [-o] [-c codepage] [-a] [-j] [-0 to -9] [-b|-m] [-k 512] [-p pwd] [-s] file.zip [files]\n\n" \
            "  -x  Extract files\n" \
            "  -l  List files\n" \
            "  -d  Destination directory\n" \
--- minizip.pc.cmakein.orig	2019-04-08 15:26:32.000000000 -0500
+++ minizip.pc.cmakein	2019-04-28 16:43:41.000000000 -0500
@@ -1,13 +1,13 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@INSTALL_LIB_DIR@
-sharedlibdir=@INSTALL_LIB_DIR@
-includedir=@INSTALL_INC_DIR@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
 
-Name: minizip
-Description: Minizip zip file manipulation library
+Name: minizip2
+Description: Minizip2 zip file manipulation library
 Version: @VERSION@
 
 Requires: zlib
-Libs: -L${libdir} -L${sharedlibdir} -lminizip
+Libs: -L${libdir} -lminizip2
+Libs.private: -lbz2 -liconv -lz
 Cflags: -I${includedir}
